src/kernels/miopen_cstdint.hpp

```diff
@@ -30,7 +30,7 @@ typedef signed char int8_t;
 typedef unsigned char uint8_t;
 typedef signed short int16_t;
 typedef unsigned short uint16_t;
-#if HIP_PACKAGE_VERSION_FLAT >= 6001000000ULL
+#if HIP_PACKAGE_VERSION_FLAT >= 6001024000ULL
```

Comment:
Similar to https://github.com/ROCm/MIOpen/pull/2650/files#r1439946058

I am afraid this is fragile and might or might not break ...